plugins/GSdx/GSDialog.cpp

```diff
@@ -166,10 +166,18 @@ void GSDialog::SetTextAsInt(UINT id, int i)
 
 void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting>& settings, int32_t selectionValue, int32_t maxValue)
 {
+	if (settings.empty()) return;
+
 	HWND hWnd = GetDlgItem(m_hWnd, id);
 
 	SendMessage(hWnd, CB_RESETCONTENT, 0, 0);
 
+	auto is_present = [&selectionValue](const GSSetting x) { return (selectionValue == x.value); };
```

Comment:
GSSetting should be passed by reference.
The parenthesis around `selectionValue == x.value` is unnecessary.
(`selectionValue` could be passed by value `[=]`.)